contractsSol5/KyberNetwork.sol

```diff
@@ -755,7 +774,7 @@ contract KyberNetwork is Withdrawable2, Utils4, IKyberNetwork, ReentrancyGuard {
         require(tData.rateOnlyNetworkFee >= tData.input.minConversionRate, "rate < minConvRate");
 
         if (gasHelper != IGasHelper(0)) {
-            address(gasHelper).call(
+            (bool success, ) = address(gasHelper).call(
```

Comment:
Is it possible to not have the `(bool success)` and just do the function call. Since we don't care if it reverts or not.